.github/workflows/check-if-migration-is-required.yml

```diff
@@ -46,4 +46,9 @@ jobs:
           PGPASSWORD: postgres
         run: |
           # Below will exit with non-zero status if model changes are missing migrations
-          sentry django makemigrations -n ci_test --check --dry-run --no-input || (echo '::error::Error: Migration required -- to generate a migration, run `sentry django makemigrations -n <some_name>`' && exit 1)
+          sentry django makemigrations --check --dry-run --no-input
+          if [ "$?" == 1 ]; then
+            echo >&2 "::error::Error: Migration required -- to generate a migration, run `sentry django makemigrations -n <some_name> && git add migrations_lockfile.txt`" && exit 1
+          else:
+            echo >&2 "::error::Error: Migration lockfile mismatch -- run `sentry django makemigrations -n <some_name> && git add migrations_lockfile.txt`" && exit 1
```

Comment:
@wedamija I simplified the test by not discussing the reason but the outcome and steps to follow.

Reply:
👍  In the future I'll probably update the migration docs with tips here, and we can just link straight to that.